portfolio/src/main/webapp/index.html

```diff
@@ -8,21 +8,21 @@
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <link rel="stylesheet" href="style.css">
-    <link rel="icon" href="images/favicon.ico" />
+    <link rel="icon" href="images/favicon.ico">
   </head>
-  <body>
+  <body id="main-body">
```

Comment:
This `id` might not be necessary since you'll only have one `body` element on the page

Reply:
Yep, it's not necessary here, thanks!